src/python/datastore/data_types.py

```diff
@@ -190,6 +190,7 @@ class SecurityImpact(object):
   HEAD = 2
   NONE = 3
   MISSING = 4
+  EXTENDED_STABLE = 5
```

Comment:
nit: put this as the first value. since these values aren't actually stored anywhere, we can change this to make EXTENDED_STABLE = 0.